include/libdnf/common/sack/query_cmp.hpp

```diff
@@ -0,0 +1,81 @@
+/*
+Copyright (C) 2020 Red Hat, Inc.
+
+This file is part of libdnf: https://github.com/rpm-software-management/libdnf/
+
+Libdnf is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+Libdnf is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#ifndef LIBDNF_COMMON_SACK_QUERY_CMP_HPP
+#define LIBDNF_COMMON_SACK_QUERY_CMP_HPP
+
+
+#include <cstdint>
+#include <type_traits>
+
+
+namespace libdnf::sack {
+
+
+// https://docs.djangoproject.com/en/3.0/ref/models/querysets/#field-lookups
+enum class QueryCmp : uint32_t {
+    // MODIFIERS
+    NOT = (1 << 0),    // negation
```

Comment:
Believe that we have to create more combination with not.

I am also not sure what is valid for version o release? It is a string but compared like a number.